std/net/curl.d

```diff
@@ -2330,9 +2330,9 @@ private bool decodeLineInto(Terminator, Char = char)(ref const(ubyte)[] basesrc,
   * http.perform();
   *
   * // Track progress
+  * auto http = HTTP();
   * http.method = HTTP.Method.get;
-  * http.url = "http://upload.wikimedia.org/wikipedia/commons/"
-  *            "5/53/Wikipedia-logo-en-big.png";
+  * http.url = "http://upload.wikimedia.org/wikipedia/commons/5/53/Wikipedia-logo-en-big.png";
```

Comment:
With concatination:
![newfigy](https://user-images.githubusercontent.com/21064622/63337469-b2204900-c349-11e9-9e36-7dbbd8c70bee.gif)

Without
![figy](https://user-images.githubusercontent.com/21064622/63337493-bc424780-c349-11e9-8315-807de166ce94.gif)

Reply:
I'm not sure what you're trying to show here. The generated code should be identical, and even if it weren't, the concatenation happens only once and will take an unmeasurable amount of time. Any perceived difference in speed will be due to fluctuations in your network connection.

Reply:
Well, I'm trying to say that new concatination (`~`) completely breaks the functionality, so I removed concatination completely and made the statement in a single line. 


### Original version:
```
http.url = "http://upload.wikimedia.org/wikipedia/commons/"
                 "5/53/Wikipedia-logo-en-big.png";
```
**Shows Error:** 
`Error: Implicit string concatenation is deprecated, use "http://upload.wikimedia.org/wikipedia/commons/" ~ "5/53/Wikipedia-logo-en-big.png" instead`

---


**Rewritten Original version:** uses new concatination syntax 
```
http.url = "http://upload.wikimedia.org/wikipedia/commons/~" ~
                 "5/53/Wikipedia-logo-en-big.png";
```
**Does not work:** the output file is corrupt, download progress does not match what is downloaded.

---

**Rewritten Original version:** removed/do not use concatination 
```
http.url = "http://upload.wikimedia.org/wikipedia/commons/5/53/Wikipedia-logo-en-big.png";
```
**Works Perfectly,** downloads the file, shows correct information about file progress.

Reply:
Oh, I see.

You accidentally typed an extra `~` in your "Rewritten Original Version". Remove the `~` from before the `"` and it should work.